app/uat_actions/uat_actions/generate_plates.rb

```diff
@@ -50,6 +50,7 @@ def perform
         report["plate_#{i}"] = plate.human_barcode
       end
     end
+    true
```

Comment:
what effect did this missing return value have?

Reply:
The perform in most of the uat actions returns a boolean (usually the return from the save of the new object). The plate and tube rack performs create multiple objects so were returning a counter number from the loop they were running, not very useful.
There were no tests for these uat actions, so I added some and needed to test the perform worked. Any better idea?